cli/tests/init/python-env.exp

```diff
@@ -1,35 +0,0 @@
-# Activate a project environment using --dir and check an alias is loaded from
-# ~/.bashrc
-
-set dir [lindex $argv 0]
-set flox $env(FLOX_BIN)
-
-set timeout 30
```

Comment:
I had a brief look at this when the job failed last week. It looks like the test will always be slow(er) because the first activation requires `virtualenv` to do it's thing and then `pip` downloads some dependencies over the network. Notably it marks the test as failed after outputting `Creating python virtual environment` and before outputting `Activating python virtual environment`.

I don't expect (🥁) that removing `expect` from these tests will make them any faster or more reliable than raising this timeout would, but it should give better feedback and make them easier to reason about 👍🏻